ui/src/store.js

```diff
@@ -0,0 +1,186 @@
+import autodux from 'autodux';
+import { doFetch } from './utils/fetch-websocket';
+
+export const {
+  reducer,
+  initial: defaultState,
+  actions: {
+    activateConnection,
+    deactivateConnection,
+    serverConnected,
+    serverDisconnected,
+    updatePurses,
+    changePurse,
+    swapInputs,
+    changeAmount,
+    createOffer,
+    resetState,
+    updateInvitationDepositId,
+  },
+} = autodux({
+  slice: 'autoswap',
+  initial: {
+    active: false,
+    connected: false,
+    account: null,
+    purses: null,
+    inputPurse: null,
+    outputPurse: null,
+    inputAmount: null,
+    outputAmount: null,
+    freeVariable: null,
+    invitationDepositId: null,
+  },
+  actions: {
+    activateConnection: state => ({ ...state, active: true }),
+    deactivateConnection: state => ({ ...state, active: false }),
+    serverConnected: state => ({ ...state, connected: true }),
+    serverDisconnected: state => ({ ...state, connected: false }),
+    updatePurses: (state, purses) => ({ ...state, purses }),
+    changePurse: {
```

Comment:
Is this "change" the verb?  Or is this a purse for small change as a result of a transaction? :)

Reply:
The verb. ⛵